### PR TITLE
Dalmatian/refresh bug

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

File: server/controllers/index.js [hunk 1]
```diff
@@ -224,6 +224,16 @@ router.post('/api/voteCard', auth, (req, res) => {
     }
 });
 
+/* Player refreshes the page and asks for the current game state */
+router.get('/api/gameState', auth, (req, res) => {
+    try {
+        const currentGameState = getGameStateById(req.session.roomId).getState(req.session.user);
+        res.status(200).json({currentGameState});
+    } catch (err) { /* Player attempts to vote for a card again or game status is not appropriate */
+        res.status(400).json({ message: err.message});
```
Comment (tillywoodfield) [inline server/controllers/index.js]: `{ message: err.message }`?

File: src/components/MyCards/MyCards.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class MyCards extends React.Component {
     render() {
         return (
             <div className={styles.myCards}>
-                <CardList cards={this.props.cards} handleClick={this.props.selectCard} isEnabled={() => this.props.playCard} cy={"my-cards"} />
+                <CardList cards={this.props.cards} handleClick={this.props.selectCard} isEnabled={() => this.props.playCard && !this.props.playedCardId} cy={"my-cards"} />
```
Comment (tillywoodfield) [inline src/components/MyCards/MyCards.jsx:16]: Do we want to hide the players' cards once they've picked one?

File: src/components/shared/Auth/AuthActions.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import axios from "axios";
```
Comment (tillywoodfield) [inline src/components/shared/Auth/AuthActions.js]: Can you delete this file please

File: src/components/MyCards/MyCards.jsx (hunk 2 above)
Comment (sibela46) [reply src/components/MyCards/MyCards.jsx:16]: No? That just disables them after they have picked one, right?

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**